VirtualReality/Widgets/qMRMLVirtualRealityHomeWidget.h

```diff
@@ -66,13 +70,17 @@ public slots:
   //void updateViewFromReferenceViewCamera();
   //void setMagnificationLock(bool);
 
+  void addModuleButton(QWidget* widget, QIcon* icon);
+  void onModuleButtonPressed();
+  void onBackButtonPressed();
+
 protected slots:
   /// Update widgets from the MRML node
   void updateWidgetFromMRML();
 
 protected:
   QScopedPointer<qMRMLVirtualRealityHomeWidgetPrivate> d_ptr;
-
+  
```

Comment:
Remove spaces in line (the Slicer precommit hooks would make your commit fail if something like this was there, so it's good to pay attention).
To clarify, I don't want you to remove the line. The problem is the "trailing whitespaces"